toptal/picasso
# chore: cleanup useless tests

### Description

We don't need example stories for Form.Hint, Form.Label, Form.Error, instead it would be nice to have everything placed near Form.Field. (maybe Select/Checkbox as well)

### How to test

- check Form.Field example and make sure that Form.Error, Form.Label and Form.Hint are shown there

### Review
